### PR TITLE
Add a few more clarifications to .mailmap

### PR DESCRIPTION
**Pull request by natecook1000**
This removes a few more duplicates when generating a list of contributors.

### CODE REVIEW

**Comment by natecook1000** [comment]
@swift-ci Please smoke test

File: .mailmap [hunk 1]
```diff
@@ -33,6 +33,8 @@ Dave Abrahams <dabrahams@apple.com> <dave@Skree.local>
 Dave Abrahams <dabrahams@apple.com> <dave@Wingy.local>
 Dave Lee <davelee@lyft.com> <davelee.com@gmail.com>
 David Rönnqvist <david.ronnqvist@gmail.com> <david.ronnqvist@skype.net>
+David Rönnqvist <david.ronnqvist@gmail.com> <david.ronnqvist@gmail.com>
```
Comment (jrose-apple) [inline .mailmap]: It looks like you can just write this as `David Rönnqvist <david.ronnqvist@gmail.com>`. That makes it look less like a typo. (I had to go look up the [git-shortlog docs](https://git-scm.com/docs/git-shortlog) to see what it meant to put the same email address twice.)

**Comment by natecook1000** [comment]
@jrose-apple Thanks! What a confusing format this is! I picked up a few more of those in this round…

**Comment by natecook1000** [comment]
@swift-ci Please smoke test

**Pull request merged**